SAMv1.tex

```diff
@@ -955,13 +955,16 @@ \subsubsection{N\_CIGAR\_OP field}\label{sec:ncigar}
 expected to update {\sf n\_cigar\_op} and {\sf cigar} with the real CIGAR
 stored in the {\tt CG} tag and remove the now-redundant {\tt CG} tag.
 
-\subsubsection{SEQ encoding}\label{sec:seq}
+\subsubsection{SEQ and QUAL encoding}\label{sec:seq}
 Sequence is encoded in 4-bit values, with adjacent bases packed into the same byte starting with the highest 4 bits first.
 When {\sf l\_seq} is odd the bottom 4 bits of the last byte are undefined, but we recommend writing these as zero.
 The case-insensitive base codes `{\tt =ACMGRSVTWYHKDBN}' are mapped to $[0,15]$ respectively with all other characters mapping to `{\tt N}' (value 15).
 
 Omitted sequence, represented in SAM as `{\tt *}', is represented by {\sf l\_seq} being 0 and {\sf seq} and {\sf qual} zero-length.
 
+Base qualities are stored as bytes in the range $[0,93]$, without any +33 conversion to printable ASCII.
```

Comment:
```suggestion
Base qualities are stored as bytes in the range $[0, 93]$, without any +33 conversion to printable ASCII.
```

Reply:
Maths mode puts a nice space there (as I learned from the `$[0,15]$` in the previous paragraph).